imgix.go

```diff
@@ -153,11 +154,22 @@ func (c *Client) PathWithParams(imgPath string, params url.Values) string {
 
 	urlString += imgPath
 
+	for key, val := range params {
+		if strings.HasSuffix(key, "64") {
+			valData := []byte(val[0])
+			base64EncodedVal := base64.URLEncoding.EncodeToString(valData)
+			base64EncodedVal = strings.Replace(base64EncodedVal, "=", "", -1)
+			params.Set(key, base64EncodedVal)
+		}
+	}
```

Comment:
Could you place this in its own unexported method and provide documentation for why it's necessary?